### PR TITLE
Feature/nuc124 textview transfer

### PR DESCRIPTION
**Pull request by acsly**
Textview component transferred.

### CODE REVIEW

**Comment by acsly** [comment]
image

File: docs/components/textview.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+| Name        | Web | Mobile | Description                              |
+| ----------- | --- | ------ | ---------------------------------------- |
+| textChange  | ✔   | ✔      | Emitted when the text changes.          |
+| returnPress | ✔   | ✔      | Emitted when the return key is pressed. |
+| focus       | ✔   | ✔      | Emitted when the container is in focus. |
+| blur        | ✔   | ✔      | Emitted when the container loses focus. |
+
+
+## Live Demo
```
Comment (osmanertem) [inline docs/components/textview.md]: live demo section should be placed after description according to Yiğits latest comment

Comment (acsly) [reply docs/components/textview.md]: Moved live demo section under description.

**Comment by acsly** [comment]
image

**Comment by acsly** [comment]
image

**Pull request merged**